src/components/NewWorkspaceModal.js

```diff
@@ -154,13 +154,20 @@ const NewWorkspaceModal = Utils.withDisplayName('NewWorkspaceModal', ({
       ])]),
       h(IdContainer, [id => h(Fragment, [
         h(FormLabel, { htmlFor: id, required: true }, ['Billing project']),
+        div({ style: { margin: '0.5rem 0' } }, ['A billing project must have a billing account which Terra has access to in order to be used.']),
         h(Select, {
           id,
           isClearable: false,
           placeholder: 'Select a billing project',
           value: namespace,
           onChange: ({ value }) => setNamespace(value),
-          options: _.uniq(_.map('projectName', billingProjects)).sort()
+          options: _.uniq(_.map(({ projectName, invalidBillingAccount }) => {
```

Comment:
I was able to create a workspace in an errored billing project:
![Screen Shot 2021-09-15 at 9 48 34 AM](https://user-images.githubusercontent.com/5375000/133446416-42d41b95-a55d-492d-b3e0-37f8cc9151de.png)
![Screen Shot 2021-09-15 at 9 49 29 AM](https://user-images.githubusercontent.com/5375000/133446430-be836d14-dcf9-45a9-9437-8efae25deaca.png)
![Screen Shot 2021-09-15 at 9 50 05 AM](https://user-images.githubusercontent.com/5375000/133446459-8c9af4e0-df6f-45eb-b967-36ff6b6ef9d8.png)



```

*************************** 1. row ***************************
                               NAME: test-eh
                CROMWELL_BUCKET_URL: NULL
                    CREATION_STATUS: Error
                    BILLING_ACCOUNT: billingAccounts/00708C-45D19D-27AAFA
                            MESSAGE: project test-eh creation finished with errors: {"ResourceType":"cloudresourcemanager.v1.project","ResourceErrorCode":"403","ResourceErrorMessage":{"code":403,"message":"The caller does not have permission","status":"PERMISSION_DENIED","statusMessage":"Forbidden","requestPath":"https://cloudresourcemanager.googleapis.com/v1/projects/test-eh","httpMethod":"GET"}} - code RESOURCE_ERROR
                   CROMWELL_BACKEND: PAPIv2
                  SERVICE_PERIMETER: NULL
              GOOGLE_PROJECT_NUMBER: NULL
               INVALID_BILLING_ACCT:
               SPEND_REPORT_DATASET: NULL
                 SPEND_REPORT_TABLE: NULL
SPEND_REPORT_DATASET_GOOGLE_PROJECT: NULL
1 row in set (5.006 sec)
```

this one seems to be a regression from dev

Reply:
I think that the backend should also not allow this to happen. We should create a new ticket for this.

Reply:
This does not block errored billing projects, just ones without a valid billing account. Those are two separate things that I would strongly prefer to keep separate for PRs.

Reply:
@s-rubenstein can you link the ticket you created to address this regression?